source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -68,24 +68,33 @@ To remove a library dependency from a project, select :guilabel:`Manage Current
 Libraries
 ---------
 
-.. warning:: These are **not** links to directly plug in to the :guilabel:`VS Code` -> :guilabel:`Install New Libraries (online)` feature. Click these links to visit the vendor site to see whether they offer online installers, offline installers, or both.
+.. warning:: Many vendors (like CTRE & REV) with additional programs to aid in the use of their libraries prefer the offline installer.
```

Comment:
I think there's some missing punctuation, but I'm no English major

Reply:
@Daltz333 summoning the English major!

Reply:
If you take away the (), you are left with a broken initial sentence. I also think we should replace `prefer` with `recommend`.